src/Configuration.ts

```diff
@@ -1,14 +1,16 @@
 /**
- * Common configurations for Proxy PagoPa and external resources
+ * Common configurations for Proxy PagoPA and external resources
  */
 
 import * as t from "io-ts";
 import { WithinRangeNumber } from "italia-ts-commons/lib/numbers";
 import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
+// Localhost hostname used for debugging
```

Comment:
it's not used for debugging here, listening to localhost is the default behavior of the server, make it clear (also in the readme)

Reply:
I removed that comment, but what do you suggest to put into readme?
It's defined as the hostname, I think it's enough clear.

Reply:
just document the fact that by default the server listens on `localhost`